mono/mini/method-to-ir.c

```diff
@@ -6370,6 +6395,7 @@ static inline MonoMethod *
 mini_get_method (MonoCompile *cfg, MonoMethod *m, guint32 token, MonoClass *klass, MonoGenericContext *context)
 {
 	ERROR_DECL (error);
+	error_init_check (cfg ? &cfg->error : error);
```

Comment:
`error` will already have been initialized, why not doing the following?
```
if (cfg)
    error_init_check(&cfg->error)
```

Reply:
It is copy/pasted from the next line, on purpose, but ok either way.